js/plugins/trimTimeline.js

```diff
@@ -0,0 +1,183 @@
+window.clinicalTimeline.trimTimeline = function (maxDays, minDays, getZoomLevel, width, getTickValues, margin, formatTime, daysToTimeObject) {
+  //cuts the timeline to areas of interest
+
+  $(".x-axis").css("visibility", "hidden");
+  
+  var tolerance = (maxDays - minDays) * 0.3; //cut the timeline after how much of inactivity
```

Comment:
Maybe good if the `0.3` can be defined when calling the function. Should probably be a function on `clinicalTimeline`. One should be able to say, trim the timeline with a tolerance of 0.3, or whatever other parameter the user chooses.

I would btw not define this as a period in days, because how many pixels that represents depends on the `zoomLevel`. I think it is more comprehensible to define this as a width in pixels, possibly as a ratio of the total width of the clinicalTimeline. You can then convert that back to a number of days.